packages/gatsby/src/utils/get-static-dir.ts

```diff
@@ -1,14 +1,14 @@
-const fs = require(`fs-extra`)
-const chokidar = require(`chokidar`)
-const nodePath = require(`path`)
-const { store } = require(`../redux`)
+import fs from "fs-extra"
+import chokidar from "chokidar"
+import nodePath from "path"
+import { store } from "../redux"
 
 /**
  * copyStaticDirs
  * --
  * Copy files from the static directory to the public directory
  */
-exports.copyStaticDirs = () => {
+export const copyStaticDirs = (): void | Promise<void> => {
```

Comment:
can you look into what it would take to make this always return Promise<void> and what impacts that would have on any consuming functions? I don't like the difference in return types here

Reply:
Instead of returning `Promise.resolve`, he can just return

Reply:
Thanks @MichaelDeBoey!
I return `undefined` instead of `Promise.resolve` to pass `consistent-return` rule.